containers/sidebar/StorageSpaceStatus.js

```diff
@@ -79,7 +96,7 @@ const StorageSpaceStatus = ({ children }) => {
 };
 
 StorageSpaceStatus.propTypes = {
-    children: PropTypes.node
+    upgradeButton: PropTypes.node
```

Comment:
Should be required, right?
```suggestion
    upgradeButton: PropTypes.node.isRequired
```